src/components/Checkout/KivaCardRedemption.vue

```diff
@@ -1,9 +1,9 @@
 <template>
-	<div class="columns">
+	<div>
```

Comment:
Since there's only one child in this div we can probably remove it and just have the child be the root element of this component